components/productCardList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ProductCard from "./productCard";
+
+interface Product {
```

Comment:
`Product` interface is repeating in here. Could we export it from the Product.tsx file for better maintainability? 
This is not necessary now and more of a heads-up, but it is good think about the nature of Product entity and where we gets this data from. Once we establish api endpoints, we would likely create a type/interface for the api responses, and would use that here directly or indirectly.